docs/getting-started/installation.md

```diff
@@ -22,18 +28,31 @@ sudo apt-get install -y make gcc
 
 ## Install Go
 
-Install `go` by following the [official docs](https://golang.org/doc/install).
-Remember to set your `$PATH` environment variable, for example:
+::: tip
+**Go 1.16+** or later is required for the Cosmos SDK.
+:::
+
+Install Go one of two ways. Check out the [official docs](https://golang.org/doc/install) and go installer for the correct download for your operating system. Alternatively, you can install Go yourself from the command line. Detailed below are standard default installation locations, but feel free to customize.
+
+**[Go Binary Downloads](https://go.dev/dl/)**
+
+**Ubuntu:**
 
+At the time of this writing, the latest release is `1.16.7`. We're going to download the tarball, extract it to `/usr/local`, and export `GOROOT` to our `$PATH`
 ```bash
-mkdir -p $HOME/go/bin
-echo "export PATH=$PATH:$(go env GOPATH)/bin" >> ~/.bash_profile
-source ~/.bash_profile
+curl -OL https://golang.org/dl/go1.16.7.linux-amd64.tar.gz
+
+sudo tar -C /usr/local -xvf go1.16.7.linux-amd64.tar.gz
+
+export PATH=$PATH:/usr/local/go/bin
+
 ```
 
-::: tip
-**Go 1.16+** or later is required for the Cosmos SDK.
-:::
+Remember to add `GOPATH` to your `$PATH` environment variable. If you're not sure where that is, run `go env GOPATH`. This will allow us to run the `gaiad` binary in the next step.
```

Comment:
```suggestion
Remember to add `GOPATH` to your `$PATH` environment variable. If you're not sure where that is, run `go env GOPATH`. This will allow us to run the `gaiad` binary in the next step. If you're not sure how to set your `$PATH` take a look at [these instructions](https://superuser.com/questions/284342/what-are-path-and-other-environment-variables-and-how-can-i-set-or-use-them).
```